package.json

```diff
@@ -163,6 +163,7 @@
     "jest": "26.6.3",
     "jest-canvas-mock": "^2.3.0",
     "jest-circus": "26.6.3",
+    "jest-fetch-mock": "^3.0.3",
```

Comment:
@mgaeta wondering what adding this library adds that we don't already have with our current Jest mocks

Reply:
[`jest-fetch-mock`](https://www.npmjs.com/package/jest-fetch-mock) mocks the builtin `fetch` command in tests. The other jest mocks can only mock HTTP request with `_request`.